apps/microtvm/reference-vm/arduino/base-box/base_box_provision.sh

```diff
@@ -45,7 +45,7 @@ ADAFRUIT_BOARDS_URL="https://adafruit.github.io/arduino-board-index/package_adaf
 ESP32_BOARDS_URL="https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_dev_index.json"
```

Comment:
can we fix version for other boards as well?

Reply:
We can't cleanly fix versions for all of the others, but using a git commit hash will work. I'll make that change.